### ISSUE TITLE
data overlapped in itself in tensorboard (validation loss)

### ISSUE THREAD

**Issue by AzizCode92** [vrenkens/nabu#50]
Hi Vincent,
I had the same issues as you mentioned in the README.md file.
Looking in the web for an answer to this, I have found that it might be related to a bug in the code (trainer.py) regarding the value of the "global_step".

Here a  [link](https://github.com/tensorflow/tensorboard/blob/1.6.0/README.md#im-seeing-data-overlapped-on-itself-what-gives) for further information regarding this issue.

**Comment by AzizCode92** [comment]
Also, I have downloaded the csv file from tensorboard and I have seen that the least 20-30 steps are always steps from the past.

**Comment by vrenkens** [comment]
Thanks!

**Comment by AzizCode92** [comment]
You're welcome ;)
